File: boilerplate/java/order-service/src/main/java/com/example/orderservice/domain/models/MfaMethod.java
package com.example.orderservice.domain.models;

/**
 * Multi-factor authentication methods.
 * 
 * String-backed enum for easy serialization to/from JSON.
 * No framework dependencies - pure domain model.
 */
public enum MfaMethod {
    TOTP("TOTP"),
    WEBAUTHN("WEBAUTHN"),
    BACKUP_CODE("BACKUP_CODE");

    private final String value;

    MfaMethod(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: tools/cookiecutter-python/{{cookiecutter.project_slug}}/src/domain/models/mfa_method.py
"""MFA method types."""

from enum import Enum


class MfaMethod(str, Enum):
    """Multi-factor authentication methods.
    
    String enum for easy serialization to/from JSON.
    """
    
    TOTP = "TOTP"
    WEBAUTHN = "WEBAUTHN"
    BACKUP_CODE = "BACKUP_CODE"
